src/frontend/components/DashboardVideoLiveRunning/index.tsx

```diff
@@ -61,10 +61,16 @@ export const DashboardVideoLiveRunning = ({
         <DashboardVideoLiveStopButton video={video} />
       </Box>
       {video.live_type === LiveModeType.JITSI && (
-        <React.Fragment>
+        <Box>
           <DashboardJoinDiscussion video={video} />
-          <Chat video={video} standalone={true} />
-        </React.Fragment>
+          <Box
+            round="6px"
+            height="700px"
+            border={{ color: 'blue', size: 'xsmall' }}
+          >
+            <Chat video={video} standalone={true} />
```

Comment:
Isn't standalone removed ?

Reply:
I simplied my commit and now standalone is not removed anymore